README.md

```diff
@@ -43,13 +45,55 @@ logger.error(`Some error while processing cool object with id ${object.id}`, { e
 ```
 
 On the console it will be logged as a json:
+
 ```sh
-{"level":"info","message":"Some info about processing cool object with id 10","extra_data":{"extra":{"data":{"id":"11","someInfo":"someInfo"}}},"logger_name":"path/to/my/file.js","timestamp":"2018-12-19T18:15:57.078Z"}
+[info] Some info about processing cool object with id 11 [trace-id: undefined] {  extra: {    extra: {      data: {        id: 11,        someInfo: 'someInfo'      }    }  },  module: 'path/to/my/file.js',  timestamp: '2020-06-08T15:32:37.472Z'}
```

Comment:
Are you sure it is printinng trace-id undefined?

Reply:
Other, I think you did a config to expose the timestamp

Reply:
> Are you sure it is printinng trace-id undefined?

It's wrong actually, I'll update it.

Reply:
> Other, I think you did a config to expose the timestamp

Yes, the `NODE_ENV` env var should be different of `production`. I'll add it to README too.